mitodl/micromasters
# Added line items to cybersource payload

#### What are the relevant tickets?
Fixes #1422

#### What's this PR do?
Passes the course key in the SKU field and the course title in the name field for the item. Note that the end user won't be able to see this data but we will get it in the postback data.

#### How should this be manually tested?
Set up the cybersource test environment variables, then buy a course run. The email data will be sent to me and we can verify that the new information is included there
